apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/index.php

```diff
@@ -35,6 +35,14 @@ function enqueue_script_and_style() {
 
 	wp_set_script_translations( 'a8c-fse-editor-site-launch-script', 'full-site-editing' );
 
+	wp_localize_script(
+		'a8c-fse-editor-site-launch-script',
+		'wpcomEditorSiteLaunch',
+		array(
+			'locale' => get_locale(),
```

Comment:
In other parts of ETK we're using using `get_iso_639_locale()` (https://github.com/automattic/wp-calypso/blob/0e11eeea2563b0d0cbf47425b7ba85be2d3eae33/apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/class-starter-page-templates.php#L250) which uses `get_blog_lang_code()` if available and then falls back to `get_locale` otherwise. Do we need to do a similar thing here? I haven't been able to find any issues with this in practice, all the locales I tried worked nicely in the launch flow.